app/javascript/components/patient/Patient.js

```diff
@@ -213,6 +217,20 @@ class Patient extends React.Component {
               {this.renderEditLink('Contact Information', 2)}
             </div>
             <div className="item-group">
+              {this.state.isMinor && (
+                <div>
+                  <span className="text-danger">Monitoree is a minor. </span>
+                  {this.props.hoh && this.props.hoh.id != this.props.details.id && (
+                    <span>
+                      View contact info for head of household (
+                      <a id="dependent-hoh-link" href={`${window.BASE_PATH}/patients/${this.props.hoh.id}`}>
```

Comment:
we use this id elsewhere, so I would remove it.  you can update the tests to `.find('a')`